platformio-tools.ini

```diff
@@ -15,26 +15,32 @@ build_unflags =
 [env:BOARD_SLIMEVR]
 platform = espressif8266 @ 4.2.1
 board = esp12e
+board_build.f_cpu = 160000000L
```

Comment:
```suggestion
```
Do not overclock official slimes. We don't know how stable it is and how it changes power consumption and wi-fi characteristics.